onnxscript/rewriter/_fusion_utils.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import onnxscript.ir as ir
+from onnxscript.rewriter import pattern
+
+
+def apply_fusion_rules(
```

Comment:
An alternative to consider: changing this as below has some advantages:
```py
def apply_fusion_rules(rules: pattern.RewriteRule | pattern.RewriteRuleSet):
   def apply_to (model: ir.Model, debug: bool = False) -> int:
       ...
   return apply_to
```
would let us define
```py
   fuse_gelu = apply_fusion_rules (gelu_rules)
```
The main advantage is that you can extend the interface by adding other options beyond debug in one place, without having to go and update every fusion file to add the same option.

Reply:
That is a good point, changed it. nit question about what the call would look like, 
For example,
`def fuse_gelu(model: ir.Model, debug: bool = False) -> int:
    fuse_gelu = _fusion_utils.apply_fusion_rules(gelu_rules)
    return fuse_gelu(model, debug=debug)
`
Would it be better name the callable function as fuse_gelu_fn or something along those lines?